README.md

```diff
@@ -9,7 +9,9 @@
 
 </div>
 
-**A versatile toolbox for 3D machine learning based on Pytorch Lightning and Hydra.**
+**A versatile framework for 3D machine learning built on Pytorch Lightning and Hydra.**
+
+Please see the [Project Guidelines](#project-guidelines) and [Contributions](#%E2%9D%A4%EF%B8%8F-contributions) sections before diving into the code!
```

Comment:
The link to contribution isn't working

Reply:
It was working before precommit came along...